libswift/Sources/Optimizer/FunctionPasses/MergeCondFails.swift

```diff
@@ -0,0 +1,94 @@
+//===--- MergeCondFail.swift -  Merge cond_fail instructions --------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SIL
+
+let mergeCondFailsPass = FunctionPass(name: "merge-condfails", runMergeCondFails)
```

Comment:
I think it's more swifty to un-abbreviate "cond" as "condition" in identifiers defined on the Swift side. E.g. `mergeCondFailsPass` → `mergeConditionFailsPass`, `runMergeCondFails` → `runMergeConditionFails`, `condFailToMerge` → `conditionFailToMerge` etc.

Reply:
In general, I agree. But in this case "CondFail" refers to the "cond_fail" instruction.